rootfs/etc/services.d/flower/run

```diff
@@ -0,0 +1,23 @@
+#!/bin/execlineb -P
+
+define user aplus
+define home /srv/${user}
+#define daemon /usr/bin/epmd
+fdmove -c 2 1
+
+# (-s is undocumented magic, that seems to move _prog_ to end of _then_ or
+# _else_ blocks and thus makes _define_ work)
+ifthenelse -s { test -e /local/venv_aplus/bin/python3 }
```

Comment:
`venv_aplus` here but in the celery run script you used `venv_${user}`. The same on the `define` line below. Maybe using the variable is a bit more generic?